.github/workflows/build-linux.yml

```diff
@@ -82,4 +82,9 @@ jobs:
       - name: Build and Test (Linux)
         if: runner.os == 'Linux'
         run: |
-          bash .github/scripts/unix.sh -t
\ No newline at end of file
+          bash .github/scripts/unix.sh -t
+      - name: Upload build directory
+        uses: actions/upload-artifact@v2
+        with:
+          name: gtsam-${{ matrix.name }}-${{ matrix.build_type }}
+          path: ${{ github.workspace }}/build/
```

Comment:
newline at end of file ?

Reply:
I checked the file again and it already has a newline.

Reply:
sorry my bad!